sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -85,24 +88,105 @@ def docker_config(self, docker_config: turing.generated.models.EnsemblerDockerCo
         else:
             self._docker_config = docker_config
 
+    @property
+    def pyfunc_config(self) -> turing.generated.models.EnsemblerPyfuncConfig:
+        return self._pyfunc_config
+
+    @pyfunc_config.setter
+    def pyfunc_config(self, pyfunc_config: turing.generated.models.EnsemblerPyfuncConfig):
+        if isinstance(pyfunc_config, turing.generated.models.EnsemblerPyfuncConfig):
```

Comment:
why not 

```python
if isinstance(pyfunc_config, dict):
            pyfunc_config['resource_request'] = \
                turing.generated.models.ResourceRequest(**pyfunc_config['resource_request'])
            self._pyfunc_config = turing.generated.models.EnsemblerPyfuncConfig(
                **pyfunc_config
            )
else
     self._pyfunc_config = pyfunc_config
```

Reply:
That's actually a really good point. I realised a lot of the setters in the SDK have been written in the same way so I'll go simplify those too.